<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSelectorDockWidget</class>
 <widget class="QDockWidget" name="DataSelectorDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Selector</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelColumns">
      <property name="text">
       <string>Columns</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textColumns">
      <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
        </sizepolicy>
      </property>     
      </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutTableAndRefresh">
      <item>
       <widget class="QComboBox" name="comboTableName">
        <property name="toolTip">
         <string>Select a source table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRefreshTables">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Refresh list of tables</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/DataSelector16.png</normaloff>../icons/DataSelector16.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelWhere">
      <property name="text">
       <string>Where</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textWhere">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
        </sizepolicy>
      </property>
      <property name="minimumSize">
        <size>
          <width>0</width>
          <height>60</height>
        </size>
      </property>
      <property name="maximumSize">
        <size>
          <width>16777215</width>
          <height>16777215</height>
        </size>
      </property>
      </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelGroupBy">
      <property name="text">
       <string>Group By</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textGroupBy">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
        </sizepolicy>
      </property>
      <property name="minimumSize">
        <size>
          <width>0</width>
          <height>60</height>
        </size>
      </property>
      <property name="maximumSize">
        <size>
          <width>16777215</width>
          <height>16777215</height>
        </size>
      </property>
      </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelOrderBy">
      <property name="text">
       <string>Order By</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textOrderBy">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
        </sizepolicy>
      </property>
      <property name="minimumSize">
        <size>
          <width>0</width>
          <height>60</height>
        </size>
      </property>
      <property name="maximumSize">
        <size>
          <width>16777215</width>
          <height>16777215</height>
        </size>
      </property>
      </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelOutputFormat">
      <property name="text">
       <string>Output Format</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboOutputFormat">
      <property name="toolTip">
       <string>Select an output format</string>
      </property>
      <item>
       <property name="text">
        <string>Shapefile</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSV file (comma delimited)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Text file (tab delimited)</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkOpenLog">
      <property name="toolTip">
       <string>Open the log file when the process is complete?</string>
      </property>
      <property name="text">
       <string>Open log after run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkClearLog">
      <property name="toolTip">
       <string>Clear the log file before the process starts?</string>
      </property>
      <property name="text">
       <string>Clear log before run</string>
      </property>
     </widget>
    </item>
    <item>
      <widget class="QLabel" name="labelMessage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutButtons">
        <item>
          <widget class="QPushButton" name="buttonLoad">
            <property name="toolTip">
              <string>Load an existing query</string>
            </property>
            <property name="text">
              <string>Load</string>
            </property>
            <property name="minimumSize">
              <size>
                <width>50</width>
                <height>0</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>50</width>
                <height>16777215</height>
              </size>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonSave">
            <property name="toolTip">
              <string>Save the query</string>
            </property>
            <property name="text">
              <string>Save</string>
            </property>
            <property name="minimumSize">
              <size>
                <width>50</width>
                <height>0</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>50</width>
                <height>16777215</height>
              </size>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonClear">
            <property name="toolTip">
              <string>Clear the form</string>
            </property>
            <property name="text">
              <string>Clear</string>
            </property>
            <property name="minimumSize">
              <size>
                <width>50</width>
                <height>0</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>50</width>
                <height>16777215</height>
              </size>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonVerify">
            <property name="toolTip">
              <string>Verify the query is valid</string>
            </property>
            <property name="text">
              <string>Verify</string>
            </property>
            <property name="minimumSize">
              <size>
                <width>50</width>
                <height>0</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>50</width>
                <height>16777215</height>
              </size>
            </property>
          </widget>
        </item>
        <item>
          <spacer name="horizontalSpacer">
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint">
              <size>
                <width>80</width>
                <height>20</height>
              </size>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QPushButton" name="buttonRun">
            <property name="toolTip">
              <string>Run the query</string>
            </property>
            <property name="text">
              <string>Run</string>
            </property>
            <property name="minimumSize">
              <size>
                <width>50</width>
                <height>0</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>50</width>
                <height>16777215</height>
              </size>
            </property>
          </widget>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="verticalSpacerBottom">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
